spec/viewAutoCompleteProvider-spec.js

```diff
@@ -24,77 +22,105 @@ function getSuggestions(prefix) {
+
+		expect(suggestions[2].type).to.equal('tag');
+		expect(suggestions[2].displayText).to.equal('Window');
+		expect(suggestions[2].snippet).to.equal('Window$1>$2</Window>');
+		expect(suggestions[2].rightLabel).to.equal('Ti.UI.Window');
+		expect(suggestions[2].description).to.equal('Ti.UI.Window: The Window is an empty drawing surface or container.');
+		expect(suggestions[2].descriptionMoreURL).to.equal('http://docs.appcelerator.com/platform/latest/#!/api/Titanium.UI.Window');
+
+		expect(suggestions[3].type).to.equal('tag');
+		expect(suggestions[3].displayText).to.equal('WindowToolbar');
+		expect(suggestions[3].snippet).to.equal('WindowToolbar$1>$2</WindowToolbar>');
+		expect(suggestions[3].rightLabel).to.equal('Ti.UI.Window.WindowToolbar');
+		expect(suggestions[3].description).to.equal('Ti.UI.Window.WindowToolbar');
+		expect(suggestions[3].descriptionMoreURL).to.equal('http://docs.appcelerator.com/platform/latest/#!/api/Titanium.UI.Window.WindowToolbar');
+	});
 });
 
 describe('Attribute suggestions', function () {
 
-	// it('should provide property suggestions', function () {
-	// 	initTextEditor('<Window s');
-	// 	suggestions = getSuggestions('status');
-
-	// 	expect(suggestions.length).toBe(7);
-
-	// 	expect(suggestions[0].type).toBe('property');
-	// 	expect(suggestions[0].displayText).toBe('scaleX');
-	// 	expect(suggestions[0].snippet).toBe('scaleX="$1"$0');
-	// 	expect(suggestions[0].rightLabel).toBe('Window');
-	// 	expect(suggestions[0].description).toBe('Ti.UI.Window: Scaling of the view in x-axis in pixels.');
-	// 	expect(suggestions[0].descriptionMoreURL).toBe('http://docs.appcelerator.com/platform/latest/#!/api/Titanium.UI.Window-property-scaleX');
-	// });
-
-	// it('should provide event suggestions for', function () {
-	// 	initTextEditor('<Window on');
-	// 	suggestions = getSuggestions('on');
-
-	// 	expect(suggestions.length).toBe(35);
-
-	// 	expect(suggestions[0].type).toBe('function');
-	// 	expect(suggestions[0].displayText).toBe('onAndroidback');
-	// 	expect(suggestions[0].snippet).toBe('onAndroidback="$1"$0');
-	// 	expect(suggestions[0].rightLabel).toBe('Window');
-	// 	expect(suggestions[0].description).toBe('Ti.UI.Window: androidback event');
-	// 	expect(suggestions[0].descriptionMoreURL).toBe('http://docs.appcelerator.com/platform/latest/#!/api/Titanium.UI.Window-event-androidback');
-	// });
+	before(async function () {
+		this.timeout(5000);
+		autoCompleteHelper.completionsFile = path.join(__dirname, 'data', 'completions');
+		atomEnvironment = global.buildAtomEnvironment();
+		await atomEnvironment.packages.activatePackage(path.join(__dirname, '..'));
+	});
+
+	after(async function () {
+		this.timeout(5000);
+		autoCompleteHelper.completionsFile = path.join(__dirname, 'data', 'completions');
+		atomEnvironment = global.buildAtomEnvironment();
+		await atomEnvironment.packages.deactivatePackage(path.join(__dirname, '..'));
+	});
+	
```

Comment:
<!--
  1 failure:  spec/viewAutoComp...
  0 warning: 
  
  
  DangerID: danger-id-default;
  File: spec/viewAutoCompleteProvider-spec.js;
  Line: 94;
-->
- :no_entry_sign: spec/viewAutoCompleteProvider-spec.js line 94 – Trailing spaces not allowed. (no-trailing-spaces)